mod/smilies.php

```diff
@@ -6,10 +6,10 @@
 use Friendica\App;
 use Friendica\Content\Smilies;
 
-function smilies_content(App $a)
+function smiliesContent(App $a)
```

Comment:
This should be kept as `smilies_content` to actually work.

Reply:
Ok changing back, but why will it not work? I'm sure it's something I'm not familiar with.. :smile: